CHANGELOG.adoc

```diff
@@ -16,6 +16,8 @@ toc::[]
 
 === Added
 
+* LegacyEncoding for iOS.
```

Comment:
Maybe this might need to be more descriptive, as more like Fixed and specifying that now sdk is also able tofilter by tags created with old ios apps

Reply:
done